src/parser/jobs/brd/PitchPerfectUsage.js

```diff
@@ -0,0 +1,314 @@
+/**
+ * @author Yumiya
+ */
+import React, {Fragment} from 'react'
+import Module from 'parser/core/Module'
+import {Accordion, Icon, Message, List} from 'semantic-ui-react'
+import STATUSES from 'data/STATUSES'
+import ACTIONS from 'data/ACTIONS'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+import styles from './Barrage.module.css'
+
+const DOT_TICK_FREQUENCY = 3000 // 3s
+const SONG_DURATION = 30000 // 30s
+const ANIMATION_LOCK = 700 // 700ms (arbitrary, fite me)
+
+const PP2_THRESHOLD = 61 // 61% crit rate
+const CONVERSION_FACTOR = 0.1
+
+// Where's the lazy scale again?
+const PP = {
+	1: 1,
+	2: 2,
+	3: 3,
+}
+
+// Issues
+const NONE = 0
+const PP2_ON_LOW_CRIT = 1
+const PP1_NOT_AT_END = 2
+
+export default class PitchPerfectUsage extends Module {
+	static handle = 'ppUsage'
+	static title = 'Pitch Perfect'
+	static dependencies = [
+		'pitchPerfect', // eslint-disable-line xivanalysis/no-unused-dependencies
```

Comment:
You don't need to depend on normalisers unless you're using a normaliser in the module. They're guaranteed to always run before the main loop.